readme.md

```diff
@@ -229,24 +229,12 @@ By default, `django-csp` is not enabled since Wagtail isn't fully compatible yet
 
 The `main` branch of this demo is designed to work with both the latest stable release and the latest `main` branch (development version) of Wagtail. To run the demo against a specific version of Wagtail, we have created [git tags](https://github.com/wagtail/bakerydemo/tags) for the latest commits that work with each feature release.
 
+- [`v6.3`](https://github.com/wagtail/bakerydemo/releases/tag/v6.3)
 - [`v6.2`](https://github.com/wagtail/bakerydemo/releases/tag/v6.2)
 - [`v6.1`](https://github.com/wagtail/bakerydemo/releases/tag/v6.1)
 - [`v6.0`](https://github.com/wagtail/bakerydemo/releases/tag/v6.0)
-- [`v5.2`](https://github.com/wagtail/bakerydemo/releases/tag/v5.2)
 
-<details>
-
-<summary>Older tags</summary>
-
-- [`v5.1`](https://github.com/wagtail/bakerydemo/releases/tag/v5.1)
-- [`v5.0`](https://github.com/wagtail/bakerydemo/releases/tag/v5.0)
-- [`v4.2`](https://github.com/wagtail/bakerydemo/releases/tag/v4.2)
-- [`v4.1`](https://github.com/wagtail/bakerydemo/releases/tag/v4.1)
-- [`v4.0`](https://github.com/wagtail/bakerydemo/releases/tag/v4.0)
-- [`v3.0`](https://github.com/wagtail/bakerydemo/releases/tag/v3.0)
-- [`v2.16`](https://github.com/wagtail/bakerydemo/releases/tag/v2.16)
-
-</details>
+See the [complete tags list](https://github.com/wagtail/bakerydemo/tags) for older releases.
```

Comment:
I figured the list can get pretty long and there's no point in listing it ourselves when GitHub already has a dedicated listing.